ext/pcntl/pcntl.stub.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+function pcntl_fork(): int {}
+
+function pcntl_waitpid(int $pid, &$status, int $options = 0, &$rusage = []): int {}
+
+function pcntl_wait(&$status, int $options = 0, &$rusage = []): int {}
+
+/**
+ * @param callable|int $handler
+ */
+function pcntl_signal(int $signo, $handler, bool $restart_syscalls = true): bool {}
+
+/** @return mixed */
+function pcntl_signal_get_handler(int $signo) {}
+
+function pcntl_signal_dispatch(): bool {}
+
+/**
+ * @param ?array $oldset
+ */
+function pcntl_sigprocmask(int $how, array $set, &$oldset = null): bool {}
+
+#ifdef HAVE_STRUCT_SIGINFO_T
+#if HAVE_SIGWAITINFO && HAVE_SIGTIMEDWAIT
+/** @return int|false */
+function pcntl_sigwaitinfo(array $set, &$info = []) {}
+
+/** @return int|false */
+function pcntl_sigtimedwait(array $set, &$info = [], int $seconds = 0, int $nanoseconds = 0) {}
+#endif
+#endif
+
+function pcntl_wifexited(int $status): bool {}
+
+function pcntl_wifstopped(int $status): bool {}
+
+#ifdef HAVE_WCONTINUED
+function pcntl_wifcontinued(int $status): bool {}
+#endif
+
+function pcntl_wifsignaled(int $status): bool {}
+
+/** @return int|false */
+function pcntl_wifexitstatus(int $status) {}
+
+/** @return int|false */
+function pcntl_wtermsig(int $status) {}
+
+/** @return int|false */
+function pcntl_wstopsig(int $status) {}
+
+/** @return null|false */
+function pcntl_exec(string $path, array $args = [], array $envs = []): ?bool {}
```

Comment:
I'm pretty confused by this function. From reading the code it looks like it *always* returns false. Do you see any case where null is actually possible?

Reply:
I came to that conclusion too, but then I assumed there was some condition where it was technically possible (for example if `try_convert_to_string()` failed but didn't throw). As you can't see anything either I'm inclined to say it's not possible so I've updated the return type to `bool` and documented it as `false`